manifest-latest.json

```diff
@@ -245,9 +245,43 @@
     "dir_name_win": "Bitcoin",
     "repo_url": "https://github.com/bitcoin/bitcoin",
     "versions": [
-      "v0.20.0"
+      "v0.20.0",
+      "v0.20.1"
     ],
-    "xbridge_conf": "bitcoin--v0.20.0.conf",
+    "xbridge_conf": "bitcoin--v0.15.1.conf",
+    "wallet_conf": "bitcoin--v0.16.0.conf"
+  },
+  {
+    "blockchain": "Bitcoin",
+    "ticker": "BTC",
+    "ver_id": "bitcoin--v0.21.0",
+    "ver_name": "Bitcoin v0.21.x",
+    "conf_name": "bitcoin.conf",
+    "dir_name_linux": "bitcoin",
+    "dir_name_mac": "Bitcoin",
+    "dir_name_win": "Bitcoin",
+    "repo_url": "https://github.com/bitcoin/bitcoin",
+    "versions": [
+      "v0.21.0",
+      "v0.21.1"
+    ],
+    "xbridge_conf": "bitcoin--v0.15.1.conf",
+    "wallet_conf": "bitcoin--v0.16.0.conf"
```

Comment:
Same issue here as mentioned above where wallet_conf & xbridge_conf file names are not consistent with `ver_id`.

Reply:
See above